Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rust:latest as builder
+
+WORKDIR ./shotover-proxy
+
+COPY ./ ./
+
+# Clean out anything already built in the target dir
+RUN rm -rf target
```

Comment:
You can add a `.dockerignore` file to exclude paths from the build context (just like `.gitignore`).